spec/integration/aws_subnet_spec.rb

```diff
@@ -41,11 +43,11 @@
             network_acl 'test_network_acl'
           end
         }.to create_an_aws_subnet('test_subnet',
```

Comment:
Thank You @tyler-ball It work for me :). 
`match_an` matcher is awesome.. Due to yesterday session & above comment I got proper clarification of the working of spec.. It really help .. Thank you.